### ISSUE TITLE
Accessibility Issues on Install Screen

### ISSUE THREAD

**Issue by RachBreeze** [umbraco/Umbraco-CMS#13283]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

v11

### Bug summary

Hi,
I have found two accessibility issues on the install screen:

- The focus state is missing off the  subscribeToNewsLetter checkbox.
- When using a screen reader, the consent for telemetry data say 2.0, rather than the displayed text.

### Steps to reproduce

- Tab to the subscribeToNewsLetter  there is not focus state on it.
- Use NVDA screen reader to listen to the value read out on the consent for telemetary data.

### Expected result / actual result

- Focus state is fixed
- The screen reader says the same thing on telemetary data as the user sees.

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/14344
